stubstub/global/setup.ts

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-console, global-require */
+// require('ts-node').register({
+// 	transpileOnly: true,
+// 	project: require('path').join(process.cwd(), 'tsconfig.server.json'),
+// });
+
+import { ChildProcessWithoutNullStreams } from 'child_process';
+import { setupTestDatabase, startTestDatabaseServer, initTestDatabase } from '../testDatabase';
+// import { sequelize } from '../../server/sequelize';
+
+// HACK(ian): The PUBPUB_SYNCING_MODELS_FOR_TEST_DB flag tells the code that we're only going to use
+// it to sync Sequelize to a test database, and that it's safe to skip certain operations.
+// In practice, we do this for the stupidest possible reason: because I can't figure out how to
+// get any modules required() from here to acknowledge absolute import paths like "server/utils"
+// instead of "../../server/utils". Since we use this absolute style extensively, we have to
+// carefully limit what we require from here. Luckily, all we need is server/models.ts, and
+// the handful of utilities that it must import.
+//
+// This is absurd, but it works. If you know how to fix this, please do!
```

Comment:
the hack is still needd, but i just set it manually below. could reinstate the function wrapper if desired, probably should just remove it